src/Chainweb/Payload/RestAPI/Server.hs

```diff
@@ -51,7 +57,25 @@ import Chainweb.Version
 import Data.CAS
 
 -- -------------------------------------------------------------------------- --
--- Handler
+-- Utils
+
+-- | The maximum number of items that are returned in a batch
+--
+payloadBatchLimit :: Int
+payloadBatchLimit = 100
+
+err404Msg :: ToJSON msg  => msg -> ServerError
+err404Msg msg = err404 { errBody = encode msg }
+
+catMaybes :: V.Vector (Maybe a) -> V.Vector a
```

Comment:
Remove please. Our cabal is >= 0.12 so this is unnecessary https://github.com/kadena-io/chainweb-node/blob/master/chainweb.cabal#L408

Reply:
It is added only in 0.12.2, which is not in our nix-pin.